include/wx/imagwebp.h

```diff
@@ -0,0 +1,62 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        wx/imagwebp.h
+// Purpose:     wxImage WebP handler
+// Author:      Hermann Höhne
+// Copyright:   (c) Hermann Höhne
+// Licence:     wxWidgets licence
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_IMAGWEBP_H_
+#define _WX_IMAGWEBP_H_
+
+#include "wx/image.h"
+
+//-----------------------------------------------------------------------------
+// wxWEBPHandler
+//-----------------------------------------------------------------------------
+
+#if wxUSE_LIBWEBP
+
+#define wxIMAGE_OPTION_WEBP_QUALITY  wxT("WebPQuality")
+#define wxIMAGE_OPTION_WEBP_LOSSLESS wxT("WebPLossless") // 0 = undefined (/mixed), 1 = lossy, 2 = lossless
+
+
+struct WXDLLIMPEXP_CORE wxWebPAnimationFrame
+{
+    wxImage image;
+    wxColour bgColour;
+    int duration;
+
+    wxWebPAnimationFrame();
+};
```

Comment:
I'd rather get rid of the trivial ctor and then avoid exporting this struct from the DLL too:

```suggestion
struct wxWebPAnimationFrame
{
    wxImage image;
    wxColour bgColour;
    int duration = 0;
};
```